kotlin/nearest-vertex/src/main/java/com/esri/arcgisruntime/sample/nearestvertex/MainActivity.kt

```diff
@@ -0,0 +1,141 @@
+/*
+ *  Copyright 2020 Esri
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package com.esri.arcgisruntime.sample.nearestvertex
+
+import android.graphics.Color
+import android.os.Bundle
+import android.view.MotionEvent
+import android.view.View.VISIBLE
+import androidx.appcompat.app.AppCompatActivity
+import com.esri.arcgisruntime.geometry.GeometryEngine
+import com.esri.arcgisruntime.geometry.Point
+import com.esri.arcgisruntime.geometry.PointCollection
+import com.esri.arcgisruntime.geometry.Polygon
+import com.esri.arcgisruntime.geometry.SpatialReferences
+import com.esri.arcgisruntime.mapping.ArcGISMap
+import com.esri.arcgisruntime.mapping.Basemap
+import com.esri.arcgisruntime.mapping.view.DefaultMapViewOnTouchListener
+import com.esri.arcgisruntime.mapping.view.Graphic
+import com.esri.arcgisruntime.mapping.view.GraphicsOverlay
+import com.esri.arcgisruntime.symbology.SimpleFillSymbol
+import com.esri.arcgisruntime.symbology.SimpleLineSymbol
+import com.esri.arcgisruntime.symbology.SimpleMarkerSymbol
+import kotlinx.android.synthetic.main.activity_main.*
+import kotlin.math.roundToInt
+
+class MainActivity : AppCompatActivity() {
+
+  override fun onCreate(savedInstanceState: Bundle?) {
+    super.onCreate(savedInstanceState)
+    setContentView(R.layout.activity_main)
+
+    // create a graphic for the polygon
+    val polygonPoints = PointCollection(SpatialReferences.getWebMercator())
+    polygonPoints.addAll(
+      listOf(
+        Point(-5991501.677830, 5599295.131468),
+        Point(-6928550.398185, 2087936.739807),
+        Point(-3149463.800709, 1840803.011362),
+        Point(-1563689.043184, 3714900.452072),
+        Point(-3180355.516764, 5619889.608838)
+      )
+    )
+    val polygon = Polygon(polygonPoints)
+    val polygonOutlineSymbol = SimpleLineSymbol(SimpleLineSymbol.Style.SOLID, Color.GREEN, 2f)
+    val polygonFillSymbol =
+      SimpleFillSymbol(SimpleFillSymbol.Style.FORWARD_DIAGONAL, Color.GREEN, polygonOutlineSymbol)
+    val polygonGraphic = Graphic(polygon, polygonFillSymbol)
+
+    // create graphics with symbols for tapped location, nearest coordinate, and nearest vertex
+    val tappedLocationGraphic = Graphic().apply {
+      symbol = SimpleMarkerSymbol(SimpleMarkerSymbol.Style.X, Color.MAGENTA, 15f)
```

Comment:
I don't know if you wanted to change the readme to match the color or vice versa